js/popup.js

```diff
@@ -118,6 +118,7 @@
                         pageInfo.tag = pageInfo.tag.concat(' ');
                     }
                     pageInfo.isPrivate = !pageInfo.shared;
+                    pageInfo.toread = (localStorage[allreadlater] === 'true');
```

Comment:
If the bookmark has been saved, you should use the saved value.

Set `pageInfo.toread` default value when `pageInfo.isSaved == false` like `pageInfo.shared`

https://github.com/lostsnow/pinboard-firefox/blob/26f44cef2026ff0c90eac6f7ca6b8c86ac907310/js/popup.js#L107-L116

Reply:
Fair point, done.